exercises/palindrome/src/example.c

```diff
@@ -0,0 +1,75 @@
+#include "palindrome.h"
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+/* Return true if the product of a and b is a palindrome, false otherwise */
+bool isPalindromeProduct(int a, int b)
+{
+   char number[10];
+   int length;
+
+   sprintf(number, "%d", a * b);
```

Comment:
By the way, you could avoid dealing with strings entirely (getting rid of several includes and speeding this up a lot) by just reversing the digits of the number and then comparing.

```
int rev(int a) {
  int ret = 0;
  while (a) {
    ret *= 10;
    ret += a%10;
    a /= 10;
  }
  return ret;
}
```
Then this function is just `return (a*b) == rev(a*b);`